### PR TITLE
Custom font size for badges

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes : #693

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:

- Variable font size
-

### CODE REVIEW

**Comment by yashLadha** [comment]
I will share the preview soon @ParthS007 :+1:

File: backend/app/generate_badges.py [hunk 1]
```diff
@@ -73,6 +77,10 @@ def generate_badges(self, aggregate, folder, index, picture, paper_size):
                                       "font-family:" + self.font_choice)
             content = content.replace("inkscape-font-specification:ubuntu",
                                       "inkscape-font-specification:" + self.font_choice)
+        print(self.font_size)
```
Comment (gabru-md) [inline backend/app/generate_badges.py]: Remove this maybe ?

Comment (yashLadha) [reply backend/app/generate_badges.py]: Yaa, it is for debugging. Will remove it.

**Comment by yashLadha** [comment]
@ParthS007 @gabru-md @vaibhavsingh97 @djmgit  You can view the changes at
https://youtu.be/lZOAHVbQrTI

**Pull request merged**